charts/audit-service/templates/deployment.yaml

```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 apiVersion: apps/v1
```

Comment:
Shouldn't this be v2

Reply:
no, it's helm that's bumped, not kubernetes